quirks/web-browser-extension-distribution-information.json

```diff
@@ -616,6 +616,25 @@
             "supported_store_identifiers": []
         },
         {
+            "long_name": "Librewolf",
+            "short_name": "Librewolf",
+            "supported_platforms": [
+                "Mac",
+                "Windows",
+                "Linux"
+            ],
+            "platform_specific_information": {
+                "Mac": {
+                    "bundle_identifier": "org.mozilla.librewolf",
+                    "code_signing_identifier": "librewolf",
+                    "code_signing_team_identifier": "not set",
```

Comment:
The fact that this browser doesn’t have a code signing team identifier means that adding it to this list won’t actually make it work with the iCloud Passwords extension. Apps need to be signed.